### PR TITLE
Cleanup formtable (/more)

### PR DESCRIPTION
**Pull request by phlax**
more cleanups/updates/fixes for formtable

### CODE REVIEW

File: pootle/core/forms.py [hunk 1]
```diff
@@ -154,23 +155,162 @@ def clean(self):
...
+    select_all_field = "select_all"
     actions = forms.ChoiceField(
         required=False,
         label=_("With selected"),
         widget=forms.Select(attrs={'class': 'js-select2'}),
-        choices=action_choices)
+        choices=(
+            ("", "----"),
+            ("reject", "Reject"),
+            ("accept", "Accept")))
```
Comment (unho) [inline pootle/core/forms.py]: Choice labels should be translatable.

File: pootle/core/forms.py [hunk 2]
```diff
@@ -154,23 +155,162 @@ def clean(self):
...
+    comment = forms.CharField(
+        label=_("add comment"),
```
Comment (unho) [inline pootle/core/forms.py]: Nit: Start with capital letter: `Add comment`.

File: pootle/static/js/common.js [hunk 3]
```diff
@@ -142,7 +142,40 @@ PTL.common = {
       }
     });
 
+    const updateFormtable = ($formtable) => {
+      $formtable.find('.js-formtable-messages').each(function () {
+        $(this).show();
+        $(this).find('ul.messages').each(function () {
+          $(this).html('<li>Updating data</li>');
```
Comment (unho) [inline pootle/static/js/common.js]: The message should be translatable.

File: pootle/core/forms.py (hunk 2 above)
Comment (phlax) [inline pootle/core/forms.py]: in this case it wants to be lower in the actual view - but ill update and make it lower in the template

**Comment by unho** [comment]
lgtm after nits

**Pull request merged**